test_runner/src/main/kotlin/ftl/config/common/CommonGcloudConfig.kt

```diff
@@ -120,6 +120,19 @@ data class CommonGcloudConfig @JsonIgnore constructor(
     @set:JsonProperty("num-flaky-test-attempts")
     var flakyTestAttempts: Int? by data
 
+    @set:CommandLine.Option(
+        names = ["--directories-to-pull"],
+        split = ",",
+        description = ["A list of paths that will be copied from the device's " +
+            "storage to the designated results bucket after the test is complete. These must be absolute paths under " +
+            "/sdcard or /data/local/tmp (for example, --directories-to-pull /sdcard/tempDir1,/data/local/tmp/tempDir2). " +
+            "Path names are restricted to the characters a-zA-Z0-9_-./+. The paths /sdcard and /data will be made available " +
+            "and treated as implicit path substitutions. E.g. if /sdcard on a particular device does not map to external " +
+            "storage, the system will replace it with the external storage path prefix for that device."]
```

Comment:
This is description for Android, not for common. Try to make it more generic

Reply:
is this a copy pasted description? if it is it should most likely be left as is.

Reply:
This is a description from android. I moved this property from android to common so now this description should have information for android and ios.